src/DynamicData.Tests/API/ApiExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.Versioning;
+using System.Threading.Tasks;
+using PublicApiGenerator;
+using VerifyXunit;
+using Xunit;
+
+namespace DynamicData.APITests;
+
+/// <summary>
+/// A helper for doing API approvals.
+/// </summary>
+[ExcludeFromCodeCoverage]
+[UsesVerify]
+public static class ApiExtensions
+{
+    /// <summary>
+    /// Checks to make sure the API is approved.
+    /// </summary>
+    /// <param name="assembly">The assembly that is being checked.</param>
+    /// <param name="namespaces">The namespaces.</param>
+    /// <param name="filePath">The caller file path.</param>
+    /// <returns>
+    /// A Task.
+    /// </returns>
+    public static async Task CheckApproval(this Assembly assembly, string[] namespaces, [CallerFilePath] string filePath = "")
```

Comment:
Why an extension method, as opposed to just putting it directly in the test? `Verifier.Verify()` has its own `[CallerFilePath]` annotation, so calling directly should still work.

Reply:
It was brought over from rxui where multiple projects and tests are run against different frameworks etc. less of an issue with dynamicdata tho.